test/lib/next-test-utils.ts

```diff
@@ -893,7 +893,7 @@ export async function hasErrorToast(
 }
 
 export async function waitForAndOpenRuntimeError(browser: BrowserInterface) {
-  return browser.waitForElementByCss('[data-nextjs-toast]').click()
+  return browser.waitForElementByCss('[data-nextjs-toast]', 5000).click()
```

Comment:
We don't need to wait 60s. If the error overlay isn't ready after 5s, it'll never be. Ideally, we'd wait for pending HMR and hydration. After that it should be there immediately.

Reply:
the default duration is 60s?

Reply:
yep. which is probably a thing to look at anyway now that you mention it.